app/views/site/edit.html.erb

```diff
@@ -1,8 +1,12 @@
 <% content_for :content do %>
   <% if Settings.status == "database_offline" or Settings.status == "api_offline" %>
+    <div class="alert alert-danger text-center">
     <p><%= t "layouts.osm_offline" %></p>
+    </div>
```

Comment:
* The contents of this `div` need to be indented. The same with other places where you are wrapping existing code with additional `divs`.